core/sql/common/OperTypeEnum.h

```diff
@@ -787,6 +787,11 @@ enum OperatorTypeEnum {
                         ITM_ISIPV6 = 2631,
                         ITM_INET_ATON = 2632,
                         ITM_INET_NTOA = 2633,
+                        ITM_CRC32     = 2634,
+                        ITM_MD5       = 2635,
+                        ITM_SHA1      = 2636,
+                        ITM_SHA2      = 2637,
+                        ITM_SHA       = 2638,
```

Comment:
ITM_SHA is never used. Looking at the parser changes, it appears that SHA is simply a syntactic sugar meaning the same as SHA1, so the difference between SHA and SHA1 disappears after parsing. So ITM_SHA is not needed.